surveys/2021-annual-survey/questions.md

```diff
@@ -219,70 +219,213 @@ TODO ask about blockers for participation?
+>
+> Previously this question used "employees" instead of "developers". It is more appropriate for us to ask about the amount
+> of developers at a company vs. the amount of people employed in total.
 
+### If you summed up the size of all Rust projects at work, how big would it be?
+
+Select one:
+- Less than 1,000 lines
+- 1,000 to 10,000 lines
+- 10,000 to 100,000 lines
+- More than 100,000 lines
+- I don't know
+
+> **REMOVED**
+>
+> While this might be interesting, there's nothing generally actionable from this. We are curious if Rust is playing a bigger
+> role in professional settings over time, and this is better served by other questions.
+
+### Is your company planning on hiring Rust developers in the next year?
+
+Select one:
+- Yes
+- No
+- I don't know
+
+> **REMOVED**
+>
+> This question was likely meant to gauge if the appetite for Rust developers is growing. However,
+> hiring is a lagging indicator, and we can more easily tell if Rust is being used more or if it is 
+> getting easier to find Rust jobs from other questions.
+
+### What are some ways you or your company is using Rust at work?
+
+Free form:
+
+> **REMOVED**
+>
+> This question is extremely vague and has not really been useful in the past.
+
+### How could we make Rust more accepted at your company?
+
+Free form:
+
+> **REMOVED**
+>
+> This question is vague and has not really been useful in the past.
 
 ## Rust in education
 
-Are you taking a course which uses or teaches Rust, or have you in the past year, or are you enrolled for one in the coming year?
+> This section is largely in service of the [contexts](./design/contexts.md) design document.
+
+### Are you taking a course or training which uses or teaches Rust, or have you in the past year, or are you enrolled for one in the coming year?
+
+Select one: 
+- Yes
+- No
 
-Where is the course or activity taught?
+### Where is the course or activity taught?
 
+Select one: 
 - University or other tertiary institute
 - Bootcamp or other vocational-focussed educational institute
 - A short training course offered by your employer or a contracted third party
 
-TODO are the latter two sufficiently differentiated?
+> TODO are the latter two sufficiently differentiated?
 
-Which best describes your course or activity?
+### Which best describes your course or activity?
 
 - A programming course which only teaches Rust
 - A programming course which teaches Rust amongst other languages
-- An operating systems course which uses Rust with or without other languages
+- An computer science course (e.g., operating systems, algorithms, etc.) course which uses Rust with or without other languages
 - Other course which uses Rust
 - Individual project
 - Group project
 - Other activity
 
-Is Rust mandated for your course or activity, or did you choose it yourself?
+### Is Rust mandated for your course or activity, or did you choose it yourself?
 
 - Compulsory
 - Optional (but suggested in some way)
```

Comment:
```suggestion
- Optional
```

It may or may not be suggested, e.g., an individual project might be allowed to be done in any language and the student chose Rust with no suggestion.